src/HMR/client/utils.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ */
+
+/* @flow */
+
+// $FlowFixMe
+import Platform from 'Platform'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Can we do `import { Platform } from 'react-native'` instead, since we discourage haste, we shouldn't be using it

It'll also remove the eslint and flow comments